src/kodetimen/playlists/3_Mellomtrinnet_5-7.txt

```diff
@@ -0,0 +1,3 @@
+code_org/codestudio_mellomtrinn.md
+../scratch/julekort/julekort.md
+processing/vesteraas.md
```

Comment:
Vi snakket litt om det, og ble enige om at vesteraas er mer lavterskel å komme i gang med, og at det er viktigere i denne settingen. Samtidig så er de nye processing-oppgavene veldig fine, slik at vi har lagt til dem som `Gå videre med Processing` for de eldste. Tanken med å bare legge disse inn på VGS var at vi ville være litt strenge på å ikke legge for mange valg i hver spilleliste, sammen med en antagelse om at de flinkeste på ungdomsskolen undersøker hva som finnes under VGS.